pkg/controller/framework/informers/factory.go

```diff
@@ -42,3 +42,19 @@ func CreateSharedPodInformer(client clientset.Interface, resyncPeriod time.Durat
 
 	return sharedInformer
 }
+
+// CreateSharedIndexPodInformer returns a SharedIndexInformer that lists and watches all pods
+func CreateSharedIndexPodInformer(client clientset.Interface, resyncPeriod time.Duration, indexers cache.Indexers) framework.SharedIndexInformer {
```

Comment:
Given the changes to the pod lister, I think I'd like this function to add the `namespace` index, so that callers don't have to think about it.